lib/functions/configuration/main-config.sh

```diff
@@ -11,6 +11,7 @@
 
 function do_main_configuration() {
 	display_alert "Starting main configuration" "${MOUNT_UUID}" "info"
+	enable_extension "disable-cnf_chroot"
```

Comment:
Hmm. An always-enabled extension in main config might not really make sense; you don't need the extension, you can simply put the 2 function/hooks directly in the core files. Even so, I'd make it conditional somehow (for example, if build host arch != target arch, and maybe only for `noble` or such if that is the case).

Reply:
Ok, thanks Ricardo, will convert and update as soon as I get home :) 
Relating to Noble, issue is really affecting all of debian; triggering condition is presence of command-not-found hook in /etc/apt/apt.conf.d

Grazie! 🙏
Ale

Reply:
Hi Guys, remove extension, integrated enablement/disablement of "command-not-found" apt hooks into main-config.sh depending on arch
if building on amd64 run as usual, if running under qemu (arm64, riscv64, armhf) then disable cnf